pkg/sql/backfill/backfill.go

```diff
@@ -468,6 +528,36 @@ func (ib *IndexBackfiller) BuildIndexEntriesChunk(
 			return nil, nil, err
 		}
 
+		iv.CurSourceRow = ib.rowVals
+
+		// For the current row, make a list of the indexes to encode entries
+		// for.
+		indexesToEncode := ib.added
+		if len(ib.predicates) > 0 {
+			indexesToEncode = make([]sqlbase.IndexDescriptor, 0, len(ib.added))
```

Comment:
This should be allocated and stored on the index builder so that we don't allocate it per row